Jenkinsfile

```diff
@@ -36,18 +36,9 @@ node {
                 println "Will set VAGRANT_RAM to ${gigsram}"
             }
 
-            // And calculate number of CPUS available
-            String numcpus = sh (
-                script: 'nproc',
-                returnStdout: true
-            )
-
-            println "Will set VAGRANT_CPUS to ${numcpus}"
-
             // Start the machine (destroy it if present) and provision it
             sh "cd ${workspace} && vagrant destroy -f || true"
             withEnv(["VAGRANT_RAM=${gigsram}",
-                     "VAGRANT_CPUS=${numcpus}",
```

Comment:
I would prefer that we dont remove this line, it might be nice to use sometimes.